packages/svelte-loader/src/LiveMode.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
```

Comment:
Could `LiveMode` be a little confusing moving forward as we are using `SanityLive` for the Live Content API stuff?

Reply:
I think this is a good point, my thoughts are that:
- Yes it could definitely be confusing!
- We already export a `useLiveMode` function in this loader and in the React loader, so the idea of "live" is already established.
- This component is really just a wrapper that calls `useLiveMode` and saves a bit of boilerplate. Is the idea of introducing a _component_ with this name the confusing part?
- I don't think we're 100% confirmed on what the new live APIs will look like, so I'm tempted to say we don't worry too much about that for now.
- I think we'll likely do a major update of the Svelte loader soonish to support Svelte 5, so we can rethink the APIs at that point.

Reply:
This makes sense! I agree that since we already have the concept of `liveMode` established, this should be all good. Let's roll with it like you say and see how things feel once the API naming is more established, etc :)